### ISSUE TITLE
Unable to find prebuilt libraries for NodeJS version 12.x

### ISSUE THREAD

**Issue by nikhilso** [stackgl/headless-gl#200]
Hello,

I tried to install the prebuilt binaries of gl 4.6.x version and unable to find the prebuilt libraries.
Here is the log image i got.
Have the new prebuilt binaries for Windows correct?

![image](https://user-images.githubusercontent.com/157522/92590744-58731f80-f2ba-11ea-9f6c-65cdf50d0e1d.png)

**Comment by clonardo** [comment]
I was also seeing this fail on Ubuntu 18.04, with Node 12.18.3 (gl is not used directly in my app, but is a dependency of brain.js, which I am actively using).

Using yarn, I was able to work around this (as far as I can tell) by adding the following to my package.json:

`
  "resolutions": {
    "gl": "4.5.2"
  }
`

Hope this helps.

**Comment by alecf** [comment]
Also seeing this on node 14, linux x64:
```
prebuild-install WARN install No prebuilt binaries found (target=14.5.0 runtime=node arch=x64 libc= platform=linux)
```

I'm thinking that prebuilt binaries haven't been pushed up yet - @dhritzkiv is there anything we can do to get the binary releases pushed to the right place?

**Comment by dhritzkiv** [comment]
Hi all. Thanks for confirming your installation problems on your preferred platforms. Looking into it now.

Curious is the fact that there are prebuilt binaries for the [4.6.0 release](https://github.com/stackgl/headless-gl/releases/tag/v4.6.0), but `prebuild-install` isn't finding them? Does anyone see something obvious that I'm missing?

**Comment by dhritzkiv** [comment]
Ah, I see it now. Module version 81 was built for, but not v83, for some reason, despite past releases including 83…

**Comment by dhritzkiv** [comment]
The culprit was lgeiger/node-abi#93 so I rolled back node-abi

Fix is in v4.6.1
